data/events.json

```diff
@@ -61,5 +61,21 @@
                 ]
             }
         ]
+    },
+    {
+        "date": "08-01-2017",
+        "items": [
+            {
+                "title": "Car bomb explosion in Bagdad",
+                "description": [
+                    "The Islamic State (IS) has claimed responsibility behind the blast.",
+                    "The bomb was set off in a vegetable market.",
+                    "11 people have been killed and injuring dozens more."
```

Comment:
11 people have been killed and injuring dozens more. 

seems strangely phrased, 

"11 people have been killed and dozens more injured."

seems more natural.